RabbitMQ.AMQP.Client/IConsumer.cs

```diff
@@ -32,14 +33,14 @@ public interface IContext
         /// This means the message has been processed and the broker can delete it.
         /// 
         /// </summary>
-        Task AcceptAsync();
+        void Accept();
```

Comment:
I can explain on a call. Basically, I was debugging a test, and noticed that the call to `Accept()` within the task was throwing an exception (which went completely un-noticed), due to the link already being closed (due to the consumer closing). This means that operations were happening out-of-order.

Since the `AMQP.Net` lib doesn't yet provide async versions of these methods, we shouldn't either. I was just being hopeful that it would "just work".